src/referencePackages/src/microsoft.build/16.10.0/Microsoft.Build.16.10.0.csproj

```diff
@@ -27,7 +27,6 @@
     <PackageReference Include="System.Memory" Version="4.5.4" />
     <PackageReference Include="System.Text.Json" Version="4.7.0" />
     <PackageReference Include="System.Threading.Tasks.Dataflow" Version="4.9.0" />
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net472" Version="1.0.2" />
```

Comment:
For my own understanding and to improve the existing tooling, why did you remove this package reference?

Reply:
We're troubleshooting trying to find a way to get the build to work. There's an issue in the build of Microsoft.Build where it's loading the Microsoft.Build.Framework assembly from microsoft.netframework.referenceassemblies.net472 instead of from the Microsoft.Build.Framework package that was just built.